integration-tests/src/test/java/oracle/kubernetes/operator/ItModelInImageConfigUpdate.java

```diff
@@ -153,6 +153,62 @@ public void testMiiConfigUpdateNonExistJdbc() throws Exception {
     LoggerHelper.getLocal().log(Level.INFO, "SUCCESS - " + testMethodName);
   }
 
+  /**
+   * Create a domain without a JDBC DS using model in image and having configmap
+   * in the domain.yaml before deploying the domain. After deploying the domain crd,
+   * create a new image with diff tag name and model files that define a JDBC DataSource
+   * and update the domain crd to change image name to reload the model,
+   * generate new config and initiate a rolling restart.
+   *
+   * @throws Exception if domain creation, config update or test veriofication fails
+   */
+  @Test
+  public void testMiiConfigUpdateNonJdbcImage() throws Exception {
```

Comment:
tests should not throw exceptions
if you think there may be an exception - write an assert to catch it

Reply:
removed "throws Exception" from all tests